ui-tests/src/test/resources/features/integrations/jira.feature

```diff
@@ -0,0 +1,217 @@
+    And fill in period input with "10" value
+    And select "Minutes" from sql dropdown
+    And click on the "Done" button
+    Then check that position of connection to fill is "Finish"
+
+    When select the "Jira Testing" connection
+    And select "Add Issue" integration action
+    And fill in values by element data-testid
+      | projectkey       | MTP                                   |
+      | issuesummary     | Test Jira Issue Created From Syndesis |
+      | issuetypeid      | Bug                                   |
+    And click on the "Done" button
+
+    When add integration step on position "0"
+    And select "Split" integration step
+    And click on the "Next" button
+
+    When add integration step on position "1"
+    And select "Data Mapper" integration step
+    And create data mapper mappings
+      | company | description |
+    And scroll "top" "right"
+    And click on the "Done" button
+
+    When click on the "Save" link
+    And set integration name "jira-add-issue"
+    And publish integration
+    Then Integration "jira-add-issue" is present in integrations list
+    And wait until integration "jira-add-issue" gets into "Running" state
+    And check that open issue with summary "Test Jira Issue Created From Syndesis" and description "Red Hat" exists
+    And close all issues with summary "Test Jira Issue Created From Syndesis" and description "Red Hat"
+
+  @jira-transition-issue
+  Scenario: Comment on a Jira issue
+
+    Given create a new jira issue in project "MTP"
+
+    When navigate to the "Home" page
+    And click on the "Create Integration" link to create a new integration.
+    Then check visibility of visual integration editor
+    And check that position of connection to fill is "Start"
+
+    When select the "PostgresDB" connection
+    And select "Periodic SQL Invocation" integration action
+    And fill in periodic query input with "select company from contact limit(1)" value
+    And fill in period input with "10" value
+    And select "Minutes" from sql dropdown
+    And click on the "Done" button
+    Then check that position of connection to fill is "Finish"
+
+    When select the "Jira Testing" connection
+    And select "Transition Issue" integration action
+    And fill in values by element data-testid
+      | issuetransitionid | 41 |
+    And fill in issuekey for previously created issue
+    And click on the "Done" button
+
+    When click on the "Save" link
+    And set integration name "jira-transition-issue"
+    And publish integration
+    Then Integration "jira-transition-issue" is present in integrations list
+    And wait until integration "jira-transition-issue" gets into "Running" state
+    And check that previously created jira is in status "Done"
+
+  @jira-retrieve-new-comments
+  Scenario: Retreive new comments from a Jira issue
+
+    Given create a new jira issue in project "MTP"
+
+    # Create integration
```

Comment:
this comment probably can be deleted as it is the only one in the whole feature file :grin: